tests/functional/api/test_query.py

```diff
@@ -7,3 +11,28 @@ def test_basic_query(eth_tester_provider):
     df = chain.blocks.query("number", "timestamp")
     assert len(df) == 4
     assert df["timestamp"][3] > df["timestamp"][2] >= df["timestamp"][1] >= df["timestamp"][0]
+    df_all = chain.blocks.query("*")
+    columns = list(df_all.columns)
+    assert [
+        "gas_data",
+        "consensus_data",
+        "hash",
+        "number",
+        "parent_hash",
+        "size",
+        "timestamp",
+    ] == columns
+
+
+def test_block_query(eth_tester_provider):
```

Comment:
We could rewrite this test and the one below as follows:

```py
@pytest.mark.parametrize(
    "query_type,incorrect_column",
    [(BlockQuery, "numbr"), (AccountQuery, "none")],
)
def test_query_incorrect_names(query_type, incorrect_column):
    # Mismatching column raises
    with pytest.raises(ValidationError) as err:
        query_type(columns=[incorrect_column])  # NOTE: fails validation before any other checks
    # error message shows mismatching columns
    assert f"Unrecognized field '{incorrect_column}'" in str(err.value)

Reply:
I think there is an issue doing this because of start and stop
BlockQuery is looking for start and stop `block`
AccountQuery is looking for start and stop `nonce`

Reply:
> I think there is an issue doing this because of start and stop BlockQuery is looking for start and stop `block` AccountQuery is looking for start and stop `nonce`

Ah yes, `start_*` had a default value, but `stop_*` does not in both cases.

Reply:
We can resolve this in a future revision